docs/platforms/android/configuration/options.mdx

```diff
@@ -268,7 +268,7 @@ Controls how many seconds to wait before shutting down. Sentry SDKs send events
 
 <ConfigKey name="enable-tracing">
 
-A boolean value, if true, transactions and trace data will be generated and captured. This will set the `traces-sample-rate` to the recommended default of 1.0 if `traces-sample-rate` is not defined. Note that `traces-sample-rate` and `traces-sampler` take precedence over this option.
```

Comment:
I think the reason we used snake_case here, because this is how we set it in the AndroidManifest. In code it's still camelCase though. I don't know if we should mention both here, but it's probably a different issue/topic to investigate in general

Reply:
yep valid point - I think it makes sense to use camelCase here because we use that for the options header (and therefore right sidebar) of all options on this page